Vendor_Policy.md

```diff
@@ -0,0 +1,403 @@
+- Commercial promotion is explicitly allowed here
+
+### #jobs-board and #jobs-chat
+
+**#jobs-board** - Post job listings here
+**#jobs-chat** - Discuss jobs, career advice, general job talk
+
+**Requirements:**
+- Post jobs relevant to Mac admins
+- Include location/remote status
+- Include salary range if required by local law (CA, CO, NYC, etc.)
+- Salary disclosure encouraged for all postings but required where legally mandated
+- Postings without legally required salary info will be removed
+- Threaded discussion about specific jobs is fine, general chatter goes to #jobs-chat
+
+**Not Allowed:**
+- Mass recruiting via DM
+- Posting outside #jobs-board
+- Reposting the same job repeatedly without updates
+
+### Social Channels
+
+- Participate as a community member, not a vendor
+- Off-topic chat is welcome
+- Don't turn social conversations into sales opportunities
+
+## Direct Messages (DMs)
+
+### Acceptable DM Use
+
+**With Permission:**
+- Following up on public conversations
+- Answering specific questions about your product
+- Scheduling demos when requested
+- Networking and relationship building
+
+**Generally OK:**
+- Responding to someone who DMed you first
+- Following up on explicit requests for information
+- Sharing pricing when asked
+
+### Unacceptable DM Use
+
+**Never OK:**
+- Cold outreach with sales pitches
+- Bulk messaging members
+- Continuing after being told no
+- Pressuring for meetings or demos
+
+**If someone says "no" in DMs:**
+- Thank them for their time
+- Stop following up
+- Don't ask "why not"
+- Move on gracefully
+
+## Special Circumstances
+
+### Sponsored Content
+
+Occasionally, MAF may approve sponsored content or partnerships. This requires:
+- Advance approval from MAF Board
+- Clear disclosure as sponsored content
+- Value for the community
+- Alignment with community values
+
+Contact: board@macadmins.org
+
+### Product Feedback and Research
+
+Vendors seeking product feedback or conducting research:
+- Request Admin approval first
+- Post in appropriate channels
+- Be transparent about your goals
+- Respect members who decline
+- Share findings with community when appropriate
+
+### Competitive Analysis
+
+**Acceptable:**
+- Honest feature comparisons when asked
+- Objective discussions of product differences
+- Acknowledging where competitors excel
+
+**Not Acceptable:**
+- Proactively disparaging competitors
+- Spreading rumors or FUD
+- Using community to attack competitors
+
+### Vendor Conflicts
+
+If two vendors from competing companies have a disagreement:
+- Keep it professional
+- Focus on facts, not attacks
+- Consider taking to DM
+- Involve Admins if needed
+- Remember: you're representing your company
+
+## Identity Verification for Vendors
+
+Admins may request verification of vendor identity to ensure authenticity. This may include:
+- LinkedIn profile confirmation
+- Company email verification
+- Video call verification (for active participants)
```

Comment:
When does this happen (is it proactive or only after problems?)
What triggers it?
Is it required for all vendors, or selective?
What happens if someone refuses?

This could be perceived as arbitrary enforcement.